data/templates/frr/daemons.frr.tmpl

```diff
@@ -0,0 +1,50 @@
+zebra=yes
```

Comment:
Can we then also drop this file https://github.com/vyos/vyos-build/blob/current/data/live-build-config/hooks/live/30-frr-configs.chroot and call the new python script here (https://github.com/vyos/vyatta-cfg/blob/current/scripts/init/vyos-router#L305) on bootup same as we do with other parts to ensure a sane default config on bootup before loading the "real" config: https://github.com/vyos/vyatta-cfg/blob/current/scripts/init/vyos-router#L319-L324

Reply:
Yes, this would be the next logical step.